sw/device/silicon_creator/lib/drivers/spi_device.h

```diff
@@ -212,6 +280,58 @@ extern const spi_device_sfdp_table_t kSpiDeviceSfdpTable;
  */
 void spi_device_init(void);
 
+/**
+ * A SPI flash command.
+ */
+typedef struct spi_device_cmd {
+  /**
+   * Command opcode.
+   *
+   * See `spi_device_opcode_t` for a list of command opcodes.
+   */
+  spi_device_opcode_t opcode;
+  /**
+   * Address.
+   *
+   * This field is set to `kSpiDeviceNoAddress` if the command does not have an
+   * address field.
+   */
+  uint32_t address;
+  /**
+   * Payload size in words.
+   *
+   * The size of the payload does not have to be a multiple of the word size but
+   * OpenTitan's flash does not support sub-word writes so we round the size up
+   * to the next word boundary.
+   */
+  size_t payload_word_count;
```

Comment:
OpenTitan's internal flash word size is not the same as a cpu word, however.  IIRC, an OT flash word is 8 bytes.

IMHO, this should be expressed in bytes, and the `payload` member should be `uint8_t` with an `alignof(4)` to ensure we can do efficient word-sized memcpy.

Reply:
Done, PTAL.